test/unit/user_test.rb

```diff
@@ -297,7 +297,7 @@ class ExistingProviderUserTest < ActiveSupport::TestCase
       # so something like:
       # Time.now.to_f #=> 1512012308.293877
       # will be saved in DB as 1512012308
-      Timecop.freeze(Time.zone.parse('2017-11-23 03:25:08 UTC +00:00'))
+      travel_to(Time.zone.parse('2017-11-23 03:25:08 UTC +00:00'))
```

Comment:
we need `travel_back` in teardown

Reply:
If I understood the documentation correctly, the time always comes back to normal after all tests anyway:

https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel_to

But I think it's a good practice anyway.